.github/actions/deploy-ubuntu/action.yml

```diff
@@ -20,12 +20,13 @@ runs:
         $SUDO rm -rf /usr/local/lib/android
         $SUDO rm -rf /usr/share/dotnet
 
-        # Allocate a swapfile on Linux as it's not enabled by default. Needed for pytorch and mkl.
-        $SUDO fallocate -l 4GB /swapfile
-        $SUDO chmod 600 /swapfile
-        $SUDO mkswap /swapfile
-        if ! $SUDO swapon /swapfile; then
-          $SUDO rm /swapfile
+        # fallocate fails on arm64 runners
```

Comment:
What kind of error are you getting?

Reply:
I took some of the code from the previous PR. If we try to run that on the arm runner, we get `fallocate: fallocate failed: Text file busy`. That might indicate that /swapfile is already being used. Maybe we just need to change the path?

Reply:
Maybe we just need to call swapoff before?

Reply:
OK, I've added that, and its working on the arm runner now